docs/proposal-002_existing-vpc.md

```diff
@@ -0,0 +1,67 @@
+# Desing Proposal #002: Existing VPC
+
```

Comment:
I should clarify the status. Having it merged doesn't mean it's final, we can refine and augment it as needed in the future.